cms/src/components/languages/CreateOrEditLanguageModel.spec.ts

```diff
@@ -52,13 +55,20 @@ describe("CreateOrEditLanguageModal.vue", () => {
                 },
             });
 
-            // Clear the input fields to simulate an empty form
             await wrapper.find("[name='languageName']").setValue("Afrikaans");
             await wrapper.find("[name='languageCode']").setValue("afr");
 
+            let groupSelector: any;
+            await waitForExpect(() => {
+                groupSelector = wrapper.findComponent(GroupSelector);
+                expect(groupSelector.exists()).toBe(true);
+            });
+
+            await groupSelector!.findComponent(ComboboxInput).setValue("Languages");
+
             // Assert that the save button is disabled
             const saveButton = wrapper.find("[data-test='save-button']");
-            expect(saveButton.attributes("disabled")).toBeUndefined();
+            expect(saveButton.attributes("disabled")).toBeFalsy();
```

Comment:
Not a fan of "falsy" as it doesn't check for an actual value - it could be `false`, `null`, `0`, etc. So I would rather put it as `toBeFalse` (or if that doesn't exist, `toBe(false)`).

Reply:
Discussed today that one can test if the disabled prop is set on the LButton component.